[2021-12-21 09:30:02,871] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.pull_from_git 2021-12-21T09:20:00+00:00 [queued]>
[2021-12-21 09:30:02,877] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.pull_from_git 2021-12-21T09:20:00+00:00 [queued]>
[2021-12-21 09:30:02,877] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 09:30:02,877] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-21 09:30:02,877] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 09:30:02,884] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): pull_from_git> on 2021-12-21T09:20:00+00:00
[2021-12-21 09:30:02,885] {standard_task_runner.py:52} INFO - Started process 115853 to run task
[2021-12-21 09:30:02,889] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'rank_update', 'pull_from_git', '2021-12-21T09:20:00+00:00', '--job-id', '1113', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/rank_update.py', '--cfg-path', '/tmp/tmpy1wra3cg', '--error-file', '/tmp/tmprd7ubyti']
[2021-12-21 09:30:02,889] {standard_task_runner.py:77} INFO - Job 1113: Subtask pull_from_git
[2021-12-21 09:30:02,926] {logging_mixin.py:104} INFO - Running <TaskInstance: rank_update.pull_from_git 2021-12-21T09:20:00+00:00 [running]> on host airflowvm
[2021-12-21 09:30:03,004] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=rank_update
AIRFLOW_CTX_TASK_ID=pull_from_git
AIRFLOW_CTX_EXECUTION_DATE=2021-12-21T09:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-21T09:20:00+00:00
[2021-12-21 09:30:03,004] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 09:30:03,005] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'git -C /home/airflow/AlgorithmStudy_211124 pull']
[2021-12-21 09:30:03,008] {subprocess.py:75} INFO - Output:
[2021-12-21 09:30:03,541] {subprocess.py:79} INFO - Already up to date.
[2021-12-21 09:30:03,542] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-21 09:30:03,565] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=rank_update, task_id=pull_from_git, execution_date=20211221T092000, start_date=20211221T093002, end_date=20211221T093003
[2021-12-21 09:30:03,582] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-21 09:30:03,586] {local_task_job.py:151} INFO - Task exited with return code 0
